pkgs/top-level/release.nix

```diff
@@ -15,8 +15,14 @@
 , limitedSupportedSystems ? [ "i686-linux" ]
   # Strip most of attributes when evaluating to spare memory usage
 ,  scrubJobs ? true
-  # Attributes passed to nixpkgs. Don't build packages marked as unfree.
-,  nixpkgsArgs ? { config = { allowUnfree = false; inHydra = true; }; }
```

Comment:
As mentioned in the commit, I'm pretty sure Hydra is not currently working as intended, since it just says `allowUnfree = false` and the unfree detection does not distinguish between unfreeRedistributable and a generic unfree license.